core/src/main/java/com/rim/IceField/Blizzard.java

```diff
@@ -11,12 +11,12 @@
     public static int numOfRounds; //Blizzard blows
     /**
      * @param players all the players in the game
-     * @param map     the map of the game
+    // * @param map     the map of the game
```

Comment:
here we display the parameters if the map is not actual anymore, you should change it to the new parameter